webpack.config.js

```diff
@@ -0,0 +1,196 @@
+const webpack = require('webpack');
+const conf = require('./tasks/config.js');
+const pkg = require('./package.json');
+const path = require('path');
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+// Doing a webpack hack for es6: https://github.com/webpack-contrib/uglifyjs-webpack-plugin/issues/33
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const FailPlugin = require('webpack-fail-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const autoprefixer = require('autoprefixer');
+
+// Our current environment
+let env = {};
+
+// Our environment aliases
+const ENV_ALIAS = {
+  PROD: [
+    'production',
+    'prod'
+  ],
+  DEV: [
+    'development',
+    'dev'
+  ],
+  TEST: [
+    'test'
+  ]
+};
+
+/**
+* Function to return if the environment matched the environment alias
+*/
+function isEnv(envAlias) {
+  // Check for default case
+  if (envAlias === ENV_ALIAS.DEV && (!env || Object.keys(env).length === 0)) {
```

Comment:
but env is a module level variable that has a default empty object on line 14 right?